README.md

```diff
@@ -130,7 +130,7 @@ do.
 There are also "minitests" which are designs which can be viewed by a human in
 Vivado to better understand how to generate more useful designs.
 
-### [Experiments](experiments)
+### [Experiments](https://github.com/SymbiFlow/prjxray/blob/master/experiments)
```

Comment:
Can we just provide a URL to the module which will prefix these bare links with?

IE We are providing the following config;
```python
    app.add_config_value(
        'recommonmark_config', {
            'github_code_repo': 'https://github.com/SymbiFlow/prjxray',
        }, True)
```

Reply:
I have solved this in a similar way by creating an initializer function in the Domain class (this because I couldn't access the `recommonmark_config` settings from the Domain class itself.